lib/elements/rebar.rb

```diff
@@ -0,0 +1,66 @@
+require './db/base'
+require './lib/materials/steel'
```

Comment:
let's always use absolute imports
```suggestion
require 'db/base'
require 'materials/steel'
```

Reply:
ok